examples/lrocnac_isis_cmp.ipynb

```diff
@@ -0,0 +1,292 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing out a USGSCSM ISD from a PDS3 LROCNAC image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pvl\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import knoten\n",
+    "import csmapi\n",
+    "\n",
+    "os.environ['ISISROOT'] = '/usgs/pkgs/isis3.8.0_RC1/install'\n",
+    "from pysis import isis\n",
+    "from pysis.exceptions import ProcessError"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Make a CSM sensor model\n",
+    "Requires TC1S2B0_01_02842S506E1942.img in data directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2019-07-16 09:29:08--  http://pdsimage.wr.usgs.gov/Missions/Lunar_Reconnaissance_Orbiter/LROC/EDR/LROLRC_0018/DATA/EXT/2013354/NAC/M1142142198RE.IMG\n",
+      "Resolving pdsimage.wr.usgs.gov (pdsimage.wr.usgs.gov)... 8.27.177.76, 2001:1900:2200:19ff::76\n",
+      "Connecting to pdsimage.wr.usgs.gov (pdsimage.wr.usgs.gov)|8.27.177.76|:80... connected.\n",
+      "HTTP request sent, awaiting response... 302 Found\n",
+      "Location: https://pdsimage2.wr.usgs.gov/Missions/Lunar_Reconnaissance_Orbiter/LROC/EDR/LROLRC_0018/DATA/EXT/2013354/NAC/M1142142198RE.IMG [following]\n",
+      "--2019-07-16 09:29:08--  https://pdsimage2.wr.usgs.gov/Missions/Lunar_Reconnaissance_Orbiter/LROC/EDR/LROLRC_0018/DATA/EXT/2013354/NAC/M1142142198RE.IMG\n",
+      "Resolving pdsimage2.wr.usgs.gov (pdsimage2.wr.usgs.gov)... 8.27.177.76, 2001:1900:2200:19ff::76\n",
+      "Connecting to pdsimage2.wr.usgs.gov (pdsimage2.wr.usgs.gov)|8.27.177.76|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 259281864 (247M) [application/octet-stream]\n",
+      "Saving to: ‘data/M1142142198RE.IMG’\n",
+      "\n",
+      "100%[======================================>] 259,281,864 13.0MB/s   in 20s    \n",
+      "\n",
+      "2019-07-16 09:29:28 (12.4 MB/s) - ‘data/M1142142198RE.IMG’ saved [259281864/259281864]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -P data http://pdsimage.wr.usgs.gov/Missions/Lunar_Reconnaissance_Orbiter/LROC/EDR/LROLRC_0018/DATA/EXT/2013354/NAC/M1142142198RE.IMG\n",
```

Comment:
We should add a -N flag to the wget command so that we don't download the image if it already exists.